web/tests/upload.test.cjs

```diff
@@ -475,6 +479,9 @@ test("uppy_events", ({override_rewire, mock_template}) => {
             getFiles() {
                 return [];
             },
+            getFile() {
+                return file;
+            },
```

Comment:
I'm not sure if this is exactly a good idea or not, some comments would be welcome!